### ISSUE TITLE
Flag to fix lint errors

### ISSUE THREAD

**Issue by wesleimp** [dotenv-linter/dotenv-linter#225]
It would be nice to have a `--fix` flag to fix all lint errors.

```sh
--fix                Automatically fix problems
--fix-type           Specify the types of fixes to apply
```

I'm not sure how to `--fix-type` could works. I don't know what is the best approach. I have considered two options
- multiple flags: `dotenv-linter --fix-type ExtraBlankLine --fix-type LowercaseKey`
- comma separator: `dotenv-linter --fix-type "ExtraBlankLine,LowercaseKey"`

**Comment by mstruebing** [comment]
What would be great and most usable of course would be to make both options work.

I'm totally for this feature, but one thing to consider here is maybe to make a backup file:
When manipulating files something can go wrong and we will have some bugs I think.
Also `.env` files are usually not version control system tracked as they contain access tokens etc.

So I would suggest when implementing this we should always make a backup file.

**Comment by mgrachev** [comment]
It's a great feature, but before we do that I'd like to think about how to implement it better.

Maybe we'll take to work this feature in the next release after [v2.1.0](https://github.com/dotenv-linter/dotenv-linter/issues/217).

**Comment by mgrachev** [comment]
This feature will be available in [v2.2.0](https://github.com/dotenv-linter/dotenv-linter/issues/238) 🎉
